examples/light-switch-app/silabs/src/BindingHandler.cpp

```diff
@@ -61,6 +61,28 @@ void ProcessOnOffUnicastBindingCommand(CommandId commandId, const EmberBindingTa
     }
 }
 
+void ProcessLevelControlUnicastBindingCommand(CommandId commandId, const EmberBindingTableEntry & binding,
+                                       Messaging::ExchangeManager * exchangeMgr, const SessionHandle & sessionHandle, 
+                                       BindingCommandData * data)
+{
+    auto onSuccess = [](const ConcreteCommandPath & commandPath, const StatusIB & status, const auto & dataResponse) {
+        ChipLogProgress(NotSpecified, "MoveToLevel command succeeds");
```

Comment:
```suggestion
        ChipLogProgress(AppLayer, "LevelControl command succeeded");
```
Can you update the prints of the OnOff commands as well?

Reply:
Updated all the prints